TwitchLib.Api.Helix.Models/Extensions/ReleaseExtensions/Component.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TwitchLib.Api.Helix.Models.Extensions.ReleaseExtensions
```

Comment:
the namespace should be ReleasedExtensions

Reply:
Renamed